CLI/commands/common/constants.js

```diff
@@ -0,0 +1,35 @@
+module.exports = Object.freeze({
+    NETWORK: undefined,
```

Comment:
I think NETWORK shouldn't be a constant. It should be a global variable, like 'Issuer' or 'defaultGasPrice'

Reply:
I didn't know in what I was thinking....